activerecord/lib/active_record/connection_adapters/sqlite3/schema_statements.rb

```diff
@@ -185,6 +193,16 @@ def quoted_scope(name = nil, type: nil)
             scope[:type] = type if type
             scope
           end
+
+          def extract_generated_type(field)
+            case field["hidden"]
+            when nil then nil
+            when 0 then nil
+            when 1 then nil
```

Comment:
```suggestion

```

`case` should return `nil` for no match anyway so it feels unnecessary to declare these matches. Unless we want to be explicit about all possible values

Reply:
Yeah, I defaulted to explicit to make code reviewer hopefully easier and clearer. Easier to remove code that is understood than add code that isn't there. 

So, I'll remove these matches now